src/utils/zendesk.ts

```diff
@@ -1,13 +1,42 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { ZENDESK_URL_PREFIXES } from "settings";
 import { loadScript } from "./loadScript";
 
 const ZENDESK_SCRIPT_SRC =
   "https://static.zdassets.com/ekr/snippet.js?key=57ad1fbc-d174-4561-8102-b3c8d98436d5";
 
 const ZENDESK_SCRIPT_ID = "ze-snippet";
+const ZENDESK_WARMUP_DELAY_MS = 6000;
 
 export const initializeZendesk = () => {
   loadScript({
     src: ZENDESK_SCRIPT_SRC,
     id: ZENDESK_SCRIPT_ID,
+    onload: hideZendeskWidget,
   });
 };
+
+const hideZendeskWidget = () => {
+  if ((window as any).zE) {
+    const showZendeskForThisPath = ZENDESK_URL_PREFIXES.find((prefix) =>
+      window.location.pathname.startsWith(prefix)
+    );
+
+    if (!showZendeskForThisPath) {
+      (window as any).zE("webWidget", "hide");
+    }
+  }
+};
```

Comment:
This logic + the value in `ZENDESK_URL_PREFIXES` seems to imply that we are only ever going to show the Zendesk widget on the admin pages? Is that the desired logic?

My understanding was that we wanted this widget to face event-going users as well, though be toggleable on/off per event (for situations where we don't intend to provide front-line support for users)

Reply:
Backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/104

Reply:
Backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/104